src/components/Settings/SettingsTab.tsx

```diff
@@ -286,6 +336,7 @@ const SettingRow = ({
   content,
   subOnly,
   helpIcon,
+  hideToggle,
```

Comment:
what is this new prop doing?

Reply:
It just hides the toggle. I used the same component but without a toggle so it looks consistent I guess.